AmazonChimeSDK/AmazonChimeSDK/device/DefaultDeviceController.swift

```diff
@@ -37,8 +37,20 @@ import AVFoundation
     public func listAudioDevices() -> [MediaDevice] {
         var inputDevices: [MediaDevice] = []
         let loudSpeaker = getSpeakerMediaDevice()
-        if let availablePort = audioSession.availableInputs {
-            inputDevices = availablePort.map { port in MediaDevice.fromAVSessionPort(port: port) }
+        if let availableInputs = audioSession.availableInputs {
+            // On iOS 15, availableInputs returns two entries for bluetooth devices.
```

Comment:
Did you check if this is known behavioral issue for iOS 15? How do you know which one to remove?

Reply:
See other comment, they work the same. I did some research, but since iOS 15 is still in preview, there is not much info available.